.github/renovate.json5

```
@@ -83,6 +83,7 @@
     "automergeSchedule": [
       "at any time"
     ],
+    "platformAutomerge": true
```

[nit] For me, the editor keeps removing trailing commas, but I find it handy to always have them as JSON5 allows that.
```suggestion
    "platformAutomerge": true,
```